### ISSUE TITLE
rTorrent not loading in dashboard

### ISSUE THREAD

**Issue by sxhartono** [Hellowlol/HTPC-Manager#598]
### Reporting Issues:

To ensure that a I has enough information to work with please include all of the following information.

**Use proper markdown syntax to structure your post (i.e. code/log in code blocks).**

**Make sure you provide the following information below:**
- [ ] Version
Git

- [ ] Branch
master2

- [ ] Commit hash
6734237c2959f5979219510a4d9a206565e037b9

- [ ] Operating system
Linux-4.14.49-unRAID-x86_64-with

- [ ] Python version
2.7.15 (default, Aug 16 2018, 14:17:09) [GCC 6.4.0]

- [ ] What you did?
HTPC Manager connects successfully to rTorrent under settings. However, when I click on rTorrent on the dashboard, I only see a rotating circle as if to indicate that it's trying to load the page.

- [ ] What happened?
When I click on rTorrent on the dashboard, I only see a rotating circle as if to indicate that it's trying to load the page.

- [ ] What you expected?
I expect to see the current files in the rTorrent queue.

- [ ] How can we reproduce your issue?
Install ruTorrent 3.8 (rTorrent 0.9.7)

- [ ] What are your (relevant) settings?
Under rTorrent, 
Enable -> Yes
Menu Name -> rTorrent
Use SSL -> No
Host -> http://192.168.1.10:9080
RPC Path -> RPC2
Username -> admin
Password -> ***

- [ ] Include a link to your **FULL** (not just a few lines!) log file that has the error. Please use [Gist](http://gist.github.com) or [Pastebin](http://pastebin.com/).

https://pastebin.com/rWkqcKQC

Close your issue when it's solved! If you found the solution yourself please comment so that others benefit from it.

#### Link to log:

https://pastebin.com/rWkqcKQC

**Comment by Hellowlol** [comment]
does the rtorrent module work?

**Comment by sxhartono** [comment]
The rTorrent module works perfectly when accessed directly.

**Comment by Hellowlol** [comment]
I meant the rtorrent module inside htpc manger?  Not the one on the dashboard or the rtorrent webui

**Comment by sxhartono** [comment]
Yes, the rTorrent module responds correctly when I set it up via Settings. However, once I enable this module and click on the rTorrent menu on the dashboard, HTPC Manager only displays a rotating circle.

**Comment by sxhartono** [comment]
<img width="446" alt="rTorrent Module" src="https://user-images.githubusercontent.com/40059982/45626018-b7d63480-bad1-11e8-97a3-7391f9ce6f97.png">
<img width="1246" alt="Rotating circle" src="https://user-images.githubusercontent.com/40059982/45626053-ce7c8b80-bad1-11e8-8c6f-670c4b121c5b.png">

**Comment by jeremysherriff** [comment]
My python skills are pretty low but the log snippet seems to indicate that the function call '.multicall()' is not a valid function.  That function is implemented in the XML-RPC library (you can see all the references to that library in the fault callback stack).

So my best guess is that the library is missing or is an incompatible version?
